diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/utils/StringToNode.kt

```diff
@@ -0,0 +1,31 @@
+package org.cqfn.diktat.ruleset.utils
+
+import com.pinterest.ktlint.core.ast.ElementType.BLOCK
+import org.jetbrains.kotlin.cli.common.environment.setIdeaIoUseFallback
+import org.jetbrains.kotlin.cli.jvm.compiler.EnvironmentConfigFiles
+import org.jetbrains.kotlin.cli.jvm.compiler.KotlinCoreEnvironment
+import org.jetbrains.kotlin.com.intellij.lang.ASTNode
+import org.jetbrains.kotlin.com.intellij.openapi.Disposable
+import org.jetbrains.kotlin.config.CompilerConfiguration
+import org.jetbrains.kotlin.psi.KtPsiFactory
+
+class StringToNode {
+
+    /**
+     * Set property
+     */
+    init {
+        setIdeaIoUseFallback()
+    }
+
+    fun createNode(text: String): ASTNode? {
+        val project = KotlinCoreEnvironment.createForProduction(
+                Disposable {},
+                CompilerConfiguration(),
+                EnvironmentConfigFiles.JVM_CONFIG_FILES
+        ).project
+        val ktPsiFactory = KtPsiFactory(project, true)
+        val node = ktPsiFactory.createBlockCodeFragment(text, null).node
+        return node.findChildByType(BLOCK)?.firstChildNode
```

Comment:
How can it be nullable? Also, I think if supplied code is not valid (i.e. cannot be parsed) we should at least log error message, or even throw an exception.